client/js/is-sum-formula.js

```diff
@@ -0,0 +1,40 @@
+const isSumFormula = function(string, numCols, numRows) {
+	// return [col, startRow, endRow] if string is a sum function
+	// otherwise return false
+
+	// string must be in form
+	// =SUM([One capital letter][1 or more digits]:[One capital letter][1 or more digits])
+	const re = /=SUM\(([A-Z]\d+):([A-Z]\d+)\)$/;
+
+	// if string is in valid format, get col letters and row numbers
+	if (string.match(re) !== null) {
+
+		const numberPattern = /\d+/g;
+		const letterPattern = /[A-Z]/g;
+
+		const colNumbers = string.slice(4) // don't get letters in "SUM"
```

Comment:
You don't need to do this.  You are matching on this `([A-Z]\d+)` -- that will capture `A` or `A15` by itself.  

Example:

![image](https://cloud.githubusercontent.com/assets/17621133/25734147/109b6fd8-3115-11e7-9340-288b11587237.png)